jose/jws.py

```diff
@@ -13,7 +13,7 @@
 from jose.utils import base64url_decode
 
 
-def sign(payload, key, headers=None, algorithm=ALGORITHMS.HS256):
+def sign(payload, key, headers=None, algorithm=ALGORITHMS.HS256, typ="JWT"):
```

Comment:
Can we use the full parameter name `type` instead of `typ`?

Reply:
AFAIK `type` is a key word in Python, but it may just be a built-in function. Either way, it's a bit impolite/bad style to shadow built-ins. I'd suggest to use something more descriptive then `xxx_type` instead.